src/snapred/backend/data/LocalDataService.py

```diff
@@ -604,6 +606,7 @@ def writeCalibrationWorkspaces(self, record: CalibrationRecord):
         for wsName in workspaces.pop(wngt.DIFFCAL_OUTPUT, []):
             # Rebuild the filename to strip any "iteration" number:
             #   * WARNING: this workaround does not work correctly if there are multiple workspaces of each "unit" type.
+            ext = ".tar"
```

Comment:
should this use the value defined in application.yml?

Reply:
updated